### PR TITLE
Changelog update

### PR DESCRIPTION
**Pull request by Vezzra**
* Added a few missing items
* Fixed typo
* Updated release date

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -205,6 +207,11 @@ Notable changes to the FreeOrion project will be documented in this file.
 - Fixed issues with AIs replaying their turns when loading a game.
 - Fixed combat log layout / scrollbar issues.
 - Fixed issues with compressed XML saves containing invalid XML.
+- Fixed issue where fleets which had never been moved since their creation were unable to enforce blockades.
+- Fixed segfault scrolling through tabs.
```
Comment (geoffthemedio) [inline ChangeLog.md]: did this occur in the previous release?

File: ChangeLog.md [hunk 2]
```diff
@@ -205,6 +207,11 @@ Notable changes to the FreeOrion project will be documented in this file.
...
+- Use correct auto aggression icon for new fleet aggression button tooltip.
+- Fixed missing combat sensor ghosts.
```
Comment (geoffthemedio) [inline ChangeLog.md]: and this?

File: ChangeLog.md (hunk 1 above)
Comment (Vezzra) [reply ChangeLog.md]: You're right, the relevant commit is e59c35a2271278c793e4de9f31057e940683330f (cherry picked as 7f63c1c697b0e9bc70e71d00ba8e0aaf2bbd67e2 in the release branch), which fixed #2133. The function containing the bug (`TabBar::RecalcLeftRightButton()`) has been added last October.

Fixed.

File: ChangeLog.md (hunk 2 above)
Comment (Vezzra) [reply ChangeLog.md]: IIRC, there has been a long-standing issue with visibility/sensor ghosts after combats in certain situations, which I thought @Dilvish-fo finally got around to fix.

I'm not 100% sure though - @Dilvish-fo? That fix is one of yours, has that bug been around pre 0.4.7?

**Pull request merged**